config/nimdoc.cfg

```diff
@@ -160,6 +174,13 @@ doc.body_toc_group = """
 doc.body_toc_group = """
 <div class="row">
   <div class="three columns">
+  <div class="theme-switch-wrapper">
+    <label class="theme-switch" for="checkbox">
+      <input type="checkbox" id="checkbox" />
+      <div class="slider round"></div>
+    </label>
+    &nbsp;&nbsp;&nbsp; <em>Dark Mode</em>
```

Comment:
Can we use CSS here instead of the `&nbsp`?

Reply:
Well yeah, it's doable but I thought ` &nbsp;&nbsp;&nbsp;` to be an easier minor-ish solution

Reply:
_to be an easier minor-ish solution_ that does the job **for now**. There are actually plans or ideas to move the whole css to dedicated stylesheets.